sdks/python/apache_beam/io/gcp/experimental/spannerio.py

```diff
@@ -1008,31 +1007,30 @@ def _reset_count(self):
     self._cells = 0
 
   def process(self, element):
-    mg_info = element.info
+    for elem in element:
```

Comment:
I've updated the batch process to support the dataflow runner. The process is almost the same as the previous commit but now I've added the `ToList()` combine transform before passing the mutation groups from to the `_BatchFn` to create batches.

Reply:
Can you clarify ? Would would Dataflow need the elements to be combined to a list ? All runners should be able to operate on a PCollection of mutation groups.

Reply:
"Which causes finish_bundle to be called multiple times" do you mean that finish_bundle will be called once per bundle ? 
This is the expected behavior and users will observe this behavior as well. Implementation should work for arbitrary bundle sizes without users having to group PCollection elements together.

Reply:
Make sense, in that case, we don't need to alter the connector code anymore, it was working as expected. Thanks, @chamikaramj for the feedback as it is always helpful.
I'll remove the changes from the spanner io connector and update the IT test code for the assertion.

Reply:
Thanks. Lemme know when this is ready for another look. Also lets trigger the IT with new changes to make sure it passes.

Reply:
Done!